package-system/python/darwin_x64/make-python.sh

```diff
@@ -0,0 +1,171 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project. For complete copyright and license terms please see the LICENSE at the root of this distribution.
+# 
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+# REQUIREMENTS:
+#  * 'git' installed and on path
+#  * 'python3' installed and on path (can be any version > 2.7 including 3.x)
+#  * xcode command line tools installed and on path ('install_name_tool' and 'otool')
```

Comment:
Lot of irony in the requirements here.
`python3` is needed to install ... `python3`